File: src/main/java/no/nav/arbeidsgiver/sykefravarsstatistikk/api/integrasjoner/kafka/KafkaStatistikkategoriTopicKey.java
package no.nav.arbeidsgiver.sykefravarsstatistikk.api.integrasjoner.kafka;

import lombok.AllArgsConstructor;
import lombok.Getter;
import no.nav.arbeidsgiver.sykefravarsstatistikk.api.statistikk.Statistikkategori;

@Getter
@AllArgsConstructor
public class KafkaStatistikkategoriTopicKey {
    Statistikkategori kategori;
    String kode;
    int kvartal;
    int årstall;
}

File: src/main/java/no/nav/arbeidsgiver/sykefravarsstatistikk/api/integrasjoner/kafka/dto/KafkaStatistikkategoriTopicKey.kt
package no.nav.arbeidsgiver.sykefravarsstatistikk.api.integrasjoner.kafka.dto

import no.nav.arbeidsgiver.sykefravarsstatistikk.api.statistikk.Statistikkategori

data class KafkaStatistikkategoriTopicKey(
    val kategori: Statistikkategori,
    val kode: String,
    val kvartal: Int,
    val årstall: Int,
)